### DISCUSSION TITLE
Fstrim question

### DISCUSSION THREAD

**Discussion by dai5m5iq** [oguzkaganeren/manjaro-starter#281] [Q&A] [answered]
@oguzkaganeren 
A question for my own understanding: My notebook uses a SSD and I deactivated fstrim service to test it but for some reason I never get to show Fstrim config in Manjaro-Starter. My filesystem is btrfs. 
Does it depend on a specific installed driver package? 

Please clarify when and how I can use this configuration in Manjaro-Starter.
Many thanks.

**Comment by oguzkaganeren** [comment, marked as answer]
Hi, can you run the below command? What is the result.
 `lsblk -oMOUNTPOINT,PKNAME -P | grep 'MOUNTPOINT="/"'`

I used the command to get the mount point then checking is it rotational with
cat /sys/block/${mountName}/queue/rotational

**Reply by dai5m5iq** [reply]
Empty result for command:
```
  ~  lsblk -oMOUNTPOINT,PKNAME -P | grep 'MOUNTPOINT="/"'                                                                    INT ✘ 
  ~    
```
For lsblk -P I get the following for the Manjaro partition
```
lsblk -P 
NAME="sda10" MAJ:MIN="8:10" RM="0" SIZE="29,4G" RO="0" TYPE="part" MOUNTPOINTS="/var/log\x0a/var/cache\x0a/home\x0a/"
```

**Reply by oguzkaganeren** [reply]
Hmm, I will remove the control. I guess each system can return a different result.

**Reply by dai5m5iq** [reply]
Shall I open a new issue for it?

**Reply by dai5m5iq** [reply]
@oguzkaganeren Thanks ☺️.
Activation works as expected now after HDD controller was removed. 
I'll mark this topic as solved.

![image](https://user-images.githubusercontent.com/111610689/213845682-8883b8a1-df86-459a-a753-5bc6f70cfcc8.png)
